README.md

```diff
@@ -1,12 +1,10 @@
-[![Build and Push](https://github.com/kserve/modelmesh-serving/actions/workflows/build-and-push.yml/badge.svg)](https://github.com/kserve/modelmesh-serving/actions/workflows/build-and-push.yml)
```

Comment:
Thanks for removing this :)
We can replace with the equivalent job of this repo but it is [currently broken](https://github.com/opendatahub-io/modelmesh-serving/actions/workflows/build-and-push.yml)

@Jooho
Side comment but we should review overall GH Action configuration for this repo, it looks completely broken at the moment https://github.com/opendatahub-io/modelmesh-serving/actions
Wdyt?
I can create a follow up ticket

Reply:
AFAIK, those are coming from upstream.
Should we really fix that? I think OpenShift-CI is running the tests.

Reply:
> To me "fix" might mean just to disable GH Action for this repo or disable the workflows.

OK, makes sense.
Let's wait for @Jooho to comment. If we agree that GHActions is not needed here, I will find out who/how can/to disable Actions (because I don't have the privs).